ui-v2/app/styles/base/color/base-variables.scss

```diff
@@ -68,6 +68,7 @@ $gray-7: #b4b8bc;
 $gray-8: #d0d2d5;
 $gray-9: #ebecee;
 $gray-10: #f3f4f6;
+$gray-010: #fbfbfc;
```

Comment:
I think this would be confusing in the future. Can we standardize all these variables to start with a `0`? For example, $gray-8 -> $gray-08

Reply:
Hmm yeah originally the `base` files where imagined to be shared between product and marketing. Luckily the marketing teams grays used a `gray-<single-digit>` naming scheme and we chose a `gray-<triple-digit>`. Originally there was talk of marketing migrating to use our gray-scale/naming instead, it just never got there.

I think I'm gonna remove all the marketing ones for the moment at least. They don't get compiled in but for folks that don't know the history here it is confusing.

Reply:
I commited the removal on the end of this here, gonna merge this!